index.html

```diff
@@ -198,9 +198,9 @@ <h2>About me</h2>
     <section class="footer">
         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         <form action="" class="footer-form">
-            <input type="text" class="full-name" placeholder="Full name">
-            <input type="email" name="" id="" class="email" placeholder="Email address">
-            <textarea class="msg" name="" id="" cols="10" rows="4" placeholder="Enter text here"></textarea>
+            <input type="text" class="full-name" placeholder="Full name" required />
+            <input type="email" name="" id="" class="email" placeholder="Email address" required />
+            <textarea class="msg" name="" id="" cols="10" rows="4" placeholder="Enter text here" required ></textarea>
         </form>
         <button type="button">Get in touch</button>
```

Comment:
- [ ] In the way that the form works properly, you need to wrap your `button` inside the `form` tag. 
- [ ] Kindly add the property `type="submit" to the button`
- [ ] As you are following a template from Figma, it's should have the requirements mentioned on the project, kindly add the properties to the button `hover` for enable(default), diable, and hover